src/reusable-blocks/class-blocks.php

```diff
@@ -16,6 +16,9 @@ public static function isReusable( array $block ) {
 	}
 
 	/**
+	 * We get block IDs recursively to find possible
+	 * nested reusable blocks.
+	 * 
```

Comment:
@strategio I think the name of the method is a bit unclear. What about `getChildrenIdsFromPost` or something like that?

Reply:
Done @andreasciamanna